### PR TITLE
chore(docs): actualize thread pool size descriptions

### LINKED ISSUE #970: Capacity planning change
Source of change:
https://github.com/questdb/questdb/pull/2392

Where:
https://questdb.io/docs/operations/capacity-planning/

### PR DESCRIPTION
**Pull request by puzpuzpuz**
Closes #970

Also, removes XFS mentioning from the guide since we don't support this FS officially.

### CODE REVIEW

File: docs/operations/capacity-planning.md [hunk 1]
```diff
@@ -145,31 +138,33 @@ QuestDB will configure a shared worker pool to handle everything except the
...
 
-$(cpuAvailable / 2) - 1 - (line.tcp.writer.worker.count)$
+$(cpuAvailable) - (line.tcp.writer.worker.count)$
+
+Minimal size of the shared worker pool is 2, even on a single core machine.
 
 #### 16 CPU Cores or less
 
 ILP I/O Worker pool is configured to use 2 CPU cores to speed up ingestion and
 the ILP Writer is using 1 core. The shared worker pool is handing everything
```
Comment (amyshwang) [inline docs/operations/capacity-planning.md:148]: I am not sure about "handing" - "handling"?
Seeing line 160, I am quite certain that it should be handling.

**Review by amyshwang — CHANGES_REQUESTED** [review]
Editorial review

File: docs/operations/capacity-planning.md [hunk 2]
```diff
@@ -145,31 +138,33 @@ QuestDB will configure a shared worker pool to handle everything except the
 InfluxDB line protocol (ILP) writer which gets a dedicated CPU core. The worker
 count is calculated as follows:
 
-$(cpuAvailable / 2) - 1 - (line.tcp.writer.worker.count)$
+$(cpuAvailable) - (line.tcp.writer.worker.count)$
```
Comment (amyshwang) [inline docs/operations/capacity-planning.md:141]: I think this means that the config needs to be updated too? 
https://questdb.io/docs/reference/configuration

image

**Review by amyshwang — COMMENTED** [review]
Tried to check the content against Config page. Not sure if my comment is correct...

File: docs/operations/capacity-planning.md (hunk 1 above)
Comment (puzpuzpuz) [reply docs/operations/capacity-planning.md]: Fixed in bdd034c7b35355a0ee06781e9e8a71ae589ead20

File: docs/operations/capacity-planning.md [hunk 3]
```diff
@@ -127,10 +122,8 @@ forecast behavior of the database.
...
 well as affinity to pin processes to specific CPUs by ID. Shared worker threads
 service SQL execution subsystems and, in the default configuration, every other
-subsystem. Except for SQL, every other subsystem can be configured to use their
-own worker threads. More information on these settings can be found on the
-[shared worker](/docs/reference/configuration#shared-worker) configuration
-page.
+subsystem. More information on these settings can be found on the
+[shared worker](/docs/reference/configuration#shared-worker) configuration page.
 
 QuestDB will allocate CPU resources differently depending on how many CPU cores
```
[the remark this answers is not in the document]
Comment (puzpuzpuz) [reply docs/operations/capacity-planning.md:128]: Fixed and removed redundant sentences in bdd034c7b35355a0ee06781e9e8a71ae589ead20

File: docs/operations/capacity-planning.md (hunk 2 above)
Comment (puzpuzpuz) [reply docs/operations/capacity-planning.md:138]: Good idea! Updated in 15e19b34b8cab629dcb9d4d8caac464590f151ee and 39ef37c0b4e9d4bc8e7fcf5b1a58dc187d2f0444

**Review by amyshwang — APPROVED** [review]

**Pull request merged**